main.go

```diff
@@ -38,6 +38,7 @@ var (
 	showVersion         = flag.Bool("version", false, "Print version information.")
 	listenAddress       = flag.String("web.listen-address", ":9091", "Address to listen on for the web interface, API, and telemetry.")
 	metricsPath         = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	prefixPath          = flag.String("web.external-url", "", "The URL under which Pushgateway is externally reachable.")
```

Comment:
"The URL under which the Pushgateway is externally reachable."

(Article "the" is missing. In README.md, we usually call it "the Pushgateway".)